### PR TITLE
Add user feedback for login

### PR DESCRIPTION
**Pull request by ericong18**
## Description
From a user perspective, it would be nice to see different "login feedbacks", specifically when the user doesn't enter a username or password. In addition, the user can tell whether or not the login is still loading via a loading spinner. Below are screenshots demonstrating these different possible scenarios.

1. The user doesn't enter a username.
Screen Shot 2019-10-12 at 1 02 00 PM

2. The user doesn't enter a password.
Screen Shot 2019-10-12 at 1 02 15 PM

3. The user clicks the "login" button, causing the loading spinner to display.
Screen Shot 2019-10-12 at 1 02 26 PM

### CODE REVIEW

File: frontend/src/components/Login/Login.css [hunk 1]
```diff
@@ -15,6 +16,10 @@
   width: 100px
 }
 
+.login__title--container {
+  width: 100%;
+}
+
```
Comment (ericong18) [inline frontend/src/components/Login/Login.css]: FYI this also fixed a resizing issue regarding the header

File: frontend/src/components/Login/Login.js [hunk 2]
```diff
@@ -25,23 +50,24 @@ class Login extends Component {
                     <img src={logo} className="login__logo"/>
                   </div>
                 </div>
-                <div className="row">
-                  <h1 className="login__title">Welcome back.</h1>
-                </div>
                 <div className="row justify-content-center">
                   <Form onSubmit={this.handleSubmit} className="login__form">
+                    <div className="login__title">Welcome back.</div>
```
Comment (ericong18) [inline frontend/src/components/Login/Login.js]: See above regarding resizing issue

File: frontend/src/components/Login/Login.css (hunk 1 above)
Comment (ericong18) [reply frontend/src/components/Login/Login.css]: nvm, removed this because it's not necessary

**Comment by jshin029** [comment]
Looks good maybe we can add invalid password and password reset in another pr.

**Comment by adialachar** [comment]
Looks good Eric. We can do the rest of the eroor handling when we set up the sign up and token stuff

**Pull request merged**